src/Nlog.RabbitMQ.Example/Program.cs

```diff
@@ -18,10 +18,22 @@ static void Main(string[] args)
 			logEventInfo.Properties["fields"] = new List<KeyValuePair<string, object>>
 			{
 				new KeyValuePair<string, object>("FieldC", "Field c")
-			};
+			};
+
+			// Provide global application properties
+			GlobalDiagnosticsContext.Set("globalFieldA", "Global Field A");
+            GlobalDiagnosticsContext.Set("globalFieldB", "Global Field B");
+
+			// Provide context properties for the current thread
+			NestedDiagnosticsLogicalContext.Push("Nested Field C");
```

Comment:
This will not work when RabbitMQTarget  is used together with `<targets async="true">`. This is why `TargetWithContext` was introduced.